### PR TITLE
Andrew/cat 130

### PR DESCRIPTION
**Pull request by fitzworkhub**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x ] Jira story/task is put into PR title
- [ x] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ x] My changes generate no new warnings
- [x ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: indico/types/submission.py [hunk 1]
```diff
@@ -32,6 +37,29 @@ class SubmissionRetries(BaseType):
     submission_id: int
 
 
+class SubmissionReviews(BaseType):
+    f"""
+    Information about a submissions Reviews.
```
Comment (goatrocks) [inline indico/types/submission.py:42]: submission's Reviews

**Review by goatrocks — APPROVED** [review]
Looks good, if you can, please run `sphinx-build docsrc docs` before merging

**Comment by goatrocks** [comment]
Oh, also in the PR title, the ticket should be [CAT-130] at the beginning.
